src/components/CohortBuilder/Summary/SurvivalChart.js

```diff
@@ -120,7 +131,10 @@ class SurvivalPlot extends React.Component {
           onMouseLeaveDonors:
             this.props.onMouseLeaveDonors && this.props.onMouseLeaveDonors.bind(this),
           onClickDonors: this.props.onClickDonors,
-          onDomainChange: newXDomain => this.updateState({ xDomain: newXDomain }),
+          onDomainChange: newXDomain => {
+            this.props.disableResetButton() && this.props.disableResetButton.bind(this);
```

Comment:
Can you explain what this line does?

Reply:
to disable  reset zoom button , I think is clear

Reply:
Yeah, what it does its clear.

What preocupies me is the second part of the line: 
` && this.props.disableResetButton.bind(this);`

From what I understand, `this.props.disableResetButton()` will be `handleDisableResetButton` at line #298 which never returns any value:
```
  handleDisableResetButton = () => {
    this.setState({ zoomDisabled: false });
  };
```

So why the ` this.props.disableResetButton() && this.props.disableResetButton.bind(this);` and not just `this.props.disableResetButton()`?